Sources/SwiftProtobuf/AnyMessageStorage.swift

```diff
@@ -162,6 +162,36 @@ internal class AnyMessageStorage {
     throw AnyUnpackError.malformedAnyField
   }
 
+
```

Comment:
Just checking since it's hard to tell in github's diff UI, this is just a move within the file?